src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1491,6 +1491,7 @@ internal T[] GetMemberList(MemberListType listType, string? name, CacheType cach
             private string? m_defaultMemberName;
             private object? m_genericCache; // Generic cache for rare scenario specific data. It is used to cache Enum names and values.
             private object[]? _emptyArray; // Object array cache for Attribute.GetCustomAttributes() pathological no-result case.
+            private FunctionPointerInfo? _functionPointerInfo;
```

Comment:
This can be stored in `m_genericCache`.

Reply:
Thanks. I didn't want to increase the size of Type...